src/uct/api/uct.h

```diff
@@ -503,6 +504,7 @@ void uct_release_tl_resource_list(uct_tl_resource_desc_t *resources);
  * @param [out] worker_p      Filled with a pointer to the worker object.
  */
 ucs_status_t uct_worker_create(ucs_async_context_t *async,
+                               ucs_worker_param_t *param,
```

Comment:
as we discussed earlier, need to pass cpu number parameter to uct_iface_open instead